src/Aquifer.API/Endpoints/Resources/Content/SendForPublisherReview/Community/Endpoint.cs

```diff
@@ -0,0 +1,65 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.Content.SendForPublisherReview.Community;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService, IResourceHistoryService historyService) 
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Post("/resources/content/{ContentId}/send-for-publisher-review-community", "/resources/content/send-for-publisher-review-community");
```

Comment:
I don't think we'll need the second arg here. That one is specifically for managers to use to bulk send items to publisher review and won't be needed for this one